components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -43,6 +43,9 @@ export class ProjectListComponent implements OnInit {
   public applyRulesButtonText$: Observable<string>;
   public ApplyRulesStatusState = ApplyRulesStatusState;
 
+  private POLLING_INTERVAL_IN_SECONDS = 10;
```

Comment:
For the projects filter's actions, we do the polling in effects, `components/automate-ui/src/app/entities/projects/project.effects.ts` -- I'd prefer we sticked to some convention 😃Any reason this approach is blocked from us here?

💡 We only want to poll while on this page. That's probably the difference. OK then.

Reply:
Asked and answered. Exactly so. 😁 😁